Gemfile

```diff
@@ -48,7 +48,7 @@ gem 'rake', '~> 12'
 
 gem 'mime-types', "~> 3"
 
-gem 'capybara', '~> 2.13', require: false
+gem 'capybara', '>= 2.13', '< 4.0', require: false
```

Comment:
Checked locally, `rake` is green with the latest Capybara 3.x.

Changes:
1. Capybara 3 changes the default server to Puma ([doc](https://github.com/teamcapybara/capybara/blob/master/UPGRADING.md#server))
This is pretty much up to the users, `rspec-rails` don't depend on either.

2. Finders ([doc](https://github.com/teamcapybara/capybara/blob/master/UPGRADING.md#finders))
Up to the users as well, we don't wrap `first`/`all`.

3. Node ([doc](https://github.com/teamcapybara/capybara/blob/master/UPGRADING.md#node))
We don't wrap those methods, so it's up to the users.

Things might go wrong for users that would `bundle update rspec-rails` without the `--conservative` flag, they would have to deal with those update issues. However, this will only happen when Rails hits 6.1 (or when Rails will decide to bump Capybara to 3.x), and Rails' 6.0 -> 6.1 dos should pretty much cover it.